src/CSEjector.sol

```diff
@@ -63,7 +63,7 @@ contract CSEjector is
     function ejectBadPerformer(
         uint256 nodeOperatorId,
         uint256 keyIndex,
-        uint256 strikesCount
+        uint256 strikes
```

Comment:
But why?

Reply:
It's not a count, but some value. To avoid confusion, I decided to keep it just strikes, and if someone is curious, they will take a look into the nat spec and figure it out for themselves.